### ISSUE TITLE
[Docker] CLI outputs "return in set Commander"

### ISSUE THREAD

**Issue by hummal** [directus/directus#14812]
### Describe the Bug

When starting the application in docker I get new CLI outout since the latest version.
This seems more like debugging output than useful information. 
Also it is only about the caching adapter. 

<img width="714" alt="image" src="https://user-images.githubusercontent.com/633711/182568601-75e9352d-72c5-41cf-8ea0-1c6767e54bdd.png">

Also some empty return statements are written
<img width="440" alt="image" src="https://user-images.githubusercontent.com/633711/182568820-4884280c-d67c-4f72-af5e-be72e5f69001.png">

### To Reproduce

docker-compose.yml
```yml
version: '3.8'

services:
  database:
    container_name: database
    image: postgis/postgis:13-master
    volumes:
      - planter_data:/var/lib/postgresql/data
    networks:
      - directus
    environment:
      POSTGRES_USER: 'directus'
      POSTGRES_PASSWORD: 'directus'
      POSTGRES_DB: 'directus'
    healthcheck:
      test: ["CMD-SHELL", "pg_isready -U postgres"]
      interval: 5s
      timeout: 5s
      retries: 5
    logging:
      driver: "local"
      options:
        max-file: "2"
        max-size: 5m

  cache:
    container_name: cache
    image: redis:6
    networks:
      - directus
    logging:
      driver: "local"
      options:
        max-file: "2"
        max-size: 1m

  directus:
    container_name: directus
    image: directus/directus:9
    ports:
      - ${DIRECTUS_PORT}:${DIRECTUS_PORT}
    volumes:
      # By default, uploads are stored in /directus/uploads
      # Always make sure your volumes matches the storage root when using
      # local driver
      - planter_uploads:/directus/uploads
      # Make sure to also mount the volume when using SQLite
      # - ./database:/directus/database
      # If you want to load extensions from the host
      - ./admin/extensions:/directus/extensions
    networks:
      - directus
    depends_on:
      cache:
        condition: service_started
      database: 
        condition: service_healthy
    env_file: ./.env
    environment:
      NODE_ENV: ${DIRECTUS_NODE_ENV}
      PORT: ${DIRECTUS_PORT}
      KEY: ${DIRECTUS_KEY}
      SECRET: ${DIRECTUS_SECRET}
      DB_CLIENT: ${DIRECTUS_DB_CLIENT}
      DB_HOST: ${DIRECTUS_DB_HOST}
      DB_PORT: 5432
      DB_DATABASE: ${DIRECTUS_DB_DATABASE}
      DB_USER: ${DIRECTUS_DB_USER}
      DB_PASSWORD: ${DIRECTUS_DB_PASSWORD}
      CACHE_ENABLED: ${DIRECTUS_CACHE_ENABLED}
      CACHE_STORE: ${DIRECTUS_CACHE_STORE}
      CACHE_REDIS: ${DIRECTUS_CACHE_REDIS}
      CACHE_TTL: ${DIRECTUS_CACHE_TTL}
      CACHE_AUTO_PURGE: ${DIRECTUS_CACHE_AUTO_PURGE}
      ADMIN_EMAIL: ${DIRECTUS_ADMIN_EMAIL}
      ADMIN_PASSWORD: ${DIRECTUS_ADMIN_PASSWORD}
      CORS_ORIGIN: ${DIRECTUS_CORS_ORIGIN}
      CORS_ENABLED: ${DIRECTUS_CORS_ENABLED}
      DIRECTUS_USE_SSL: ${DIRECTUS_USE_SSL}
      LOG_LEVEL: ${DIRECTUS_LOG_LEVEL}
      # Make sure to set this in production
      # (see https://docs.directus.io/configuration/config-options/#general)
      PUBLIC_URL: '/'
    healthcheck:
      test: [ "CMD-SHELL", "wget http://directus/:$DIRECTUS_PORT/admin -q -O - > /dev/null 2>&1" ]
      interval: 5s
      timeout: 5s
      retries: 5
    logging:
      driver: "local"
      options:
        max-file: "2"
        max-size: 5m

networks:
  directus:

volumes:
  planter_data:
  planter_uploads:
```

.env
```
# GENERAL SECTION
NODE_ENV=development

# DIRECTUS SECTION
DIRECTUS_ACCESS_TOKEN=pYNkngRMP18gj0TNM6xC0iSpSVGRICR5
DIRECTUS_USE_SSL=false
DIRECTUS_PORT=61001
DIRECTUS_HOST=http://directus:${DIRECTUS_PORT}

DIRECTUS_KEY=255d861b-5ea1-5996-9aa3-922530ec40b1
DIRECTUS_SECRET=6116487b-cda1-52c2-b5b5-c8022c45e263
DIRECTUS_DB_CLIENT=pg
DIRECTUS_DB_HOST=database
DIRECTUS_DB_DATABASE=directus
DIRECTUS_DB_USER=directus
DIRECTUS_DB_PASSWORD=directus

DIRECTUS_CACHE_ENABLED=true
DIRECTUS_CACHE_STORE=redis
DIRECTUS_CACHE_REDIS=redis://cache:6379
DIRECTUS_CACHE_TTL=5m
DIRECTUS_CACHE_AUTO_PURGE=true
DIRECTUS_CACHE_VALUE_MAX_SIZE=100000000

DIRECTUS_ADMIN_EMAIL=test@test.com
DIRECTUS_ADMIN_PASSWORD=test

DIRECTUS_CORS_ORIGIN=true
DIRECTUS_CORS_ENABLED=true

DIRECTUS_LOG_LEVEL=warn
DIRECTUS_NODE_ENV=production

# Make sure to set this in production
# (see https://docs.directus.io/configuration/config-options/#general)
DIRECTUS_PUBLIC_URL=/
```

### Errors Shown

directus   | 08:12:08 ⚠️  PUBLIC_URL should be a full URL
directus   | return
directus   | return in set Commander {
directus   |   options: {
directus   |     host: 'cache',
directus   |     port: 6379,
directus   |     uri: 'redis://cache:6379',
directus   |     family: 4,
directus   |     connectTimeout: 10000,
directus   |     disconnectTimeout: 2000,
directus   |     retryStrategy: [Function: retryStrategy],
directus   |     keepAlive: 0,
directus   |     noDelay: true,
directus   |     connectionName: null,
directus   |     sentinels: null,
directus   |     name: null,
directus   |     role: 'master',
directus   |     sentinelRetryStrategy: [Function: sentinelRetryStrategy],
directus   |     sentinelReconnectStrategy: [Function: sentinelReconnectStrategy],
directus   |     natMap: null,
directus   |     enableTLSForSentinelMode: false,
directus   |     updateSentinels: true,
directus   |     failoverDetector: false,
directus   |     username: null,
directus   |     password: null,
directus   |     db: 0,
directus   |     enableOfflineQueue: true,
directus   |     enableReadyCheck: true,
directus   |     autoResubscribe: true,
directus   |     autoResendUnfulfilledCommands: true,
directus   |     lazyConnect: false,
directus   |     keyPrefix: '',
directus   |     reconnectOnError: null,
directus   |     readOnly: false,
directus   |     stringNumbers: false,
directus   |     maxRetriesPerRequest: 20,
directus   |     maxLoadingRetryTime: 10000,
directus   |     enableAutoPipelining: false,
directus   |     autoPipeliningIgnoredCommands: [],
directus   |     sentinelMaxConnections: 10
directus   |   },
directus   |   scriptsSet: {},
directus   |   addedBuiltinSet: Set(0) {},
directus   |   status: 'ready',
directus   |   isCluster: false,
directus   |   reconnectTimeout: null,
directus   |   connectionEpoch: 1,
directus   |   retryAttempts: 0,
directus   |   manuallyClosing: false,
directus   |   _autoPipelines: Map(0) {},
directus   |   _runningAutoPipelines: Set(0) {},
directus   |   _events: [Object: null prototype] { error: [Function (anonymous)] },
directus   |   _eventsCount: 1,
directus   |   _maxListeners: undefined,
directus   |   commandQueue: Denque {
directus   |     _capacity: undefined,
directus   |     _head: 0,
directus   |     _tail: 0,
directus   |     _capacityMask: 3,
directus   |     _list: [ undefined, undefined, undefined, undefined ]
directus   |   },
directus   |   offlineQueue: Denque {
directus   |     _capacity: undefined,
directus   |     _head: 0,
directus   |     _tail: 0,
directus   |     _capacityMask: 3,
directus   |     _list: [ <4 empty items> ]
directus   |   },
directus   |   connector: StandaloneConnector {
directus   |     connecting: true,
directus   |     disconnectTimeout: 2000,
directus   |     options: {
directus   |       host: 'cache',
directus   |       port: 6379,
directus   |       uri: 'redis://cache:6379',
directus   |       family: 4,
directus   |       connectTimeout: 10000,
directus   |       disconnectTimeout: 2000,
directus   |       retryStrategy: [Function: retryStrategy],
directus   |       keepAlive: 0,
directus   |       noDelay: true,
directus   |       connectionName: null,
directus   |       sentinels: null,
directus   |       name: null,
directus   |       role: 'master',
directus   |       sentinelRetryStrategy: [Function: sentinelRetryStrategy],
directus   |       sentinelReconnectStrategy: [Function: sentinelReconnectStrategy],
directus   |       natMap: null,
directus   |       enableTLSForSentinelMode: false,
directus   |       updateSentinels: true,
directus   |       failoverDetector: false,
directus   |       username: null,
directus   |       password: null,
directus   |       db: 0,
directus   |       enableOfflineQueue: true,
directus   |       enableReadyCheck: true,
directus   |       autoResubscribe: true,
directus   |       autoResendUnfulfilledCommands: true,
directus   |       lazyConnect: false,
directus   |       keyPrefix: '',
directus   |       reconnectOnError: null,
directus   |       readOnly: false,
directus   |       stringNumbers: false,
directus   |       maxRetriesPerRequest: 20,
directus   |       maxLoadingRetryTime: 10000,
directus   |       enableAutoPipelining: false,
directus   |       autoPipeliningIgnoredCommands: [],
directus   |       sentinelMaxConnections: 10
directus   |     },
directus   |     stream: Socket {
directus   |       connecting: false,
directus   |       _hadError: false,
directus   |       _parent: null,
directus   |       _host: 'cache',
directus   |       _readableState: [ReadableState],
directus   |       _events: [Object: null prototype],
directus   |       _eventsCount: 5,
directus   |       _maxListeners: undefined,
directus   |       _writableState: [WritableState],
directus   |       allowHalfOpen: false,
directus   |       _sockname: null,
directus   |       _pendingData: null,
directus   |       _pendingEncoding: '',
directus   |       server: null,
directus   |       _server: null,
directus   |       timeout: 0,
directus   |       [Symbol(async_id_symbol)]: 115,
directus   |       [Symbol(kHandle)]: [TCP],
directus   |       [Symbol(lastWriteQueueSize)]: 0,
directus   |       [Symbol(timeout)]: Timeout {
directus   |         _idleTimeout: -1,
directus   |         _idlePrev: null,
directus   |         _idleNext: null,
directus   |         _idleStart: 1028,
directus   |         _onTimeout: null,
directus   |         _timerArgs: undefined,
directus   |         _repeat: null,
directus   |         _destroyed: true,
directus   |         [Symbol(refed)]: false,
directus   |         [Symbol(kHasPrimitive)]: false,
directus   |         [Symbol(asyncId)]: 120,
directus   |         [Symbol(triggerId)]: 0
directus   |       },
directus   |       [Symbol(kBuffer)]: null,
directus   |       [Symbol(kBufferCb)]: null,
directus   |       [Symbol(kBufferGen)]: null,
directus   |       [Symbol(kCapture)]: false,
directus   |       [Symbol(kSetNoDelay)]: true,
directus   |       [Symbol(kSetKeepAlive)]: true,
directus   |       [Symbol(kSetKeepAliveInitialDelay)]: 0,
directus   |       [Symbol(kBytesRead)]: 0,
directus   |       [Symbol(kBytesWritten)]: 0
directus   |     }
directus   |   },
directus   |   condition: { select: 0, auth: null, subscriber: false },
directus   |   stream: <ref *1> Socket {
directus   |     connecting: false,
directus   |     _hadError: false,
directus   |     _parent: null,
directus   |     _host: 'cache',
directus   |     _readableState: ReadableState {
directus   |       objectMode: false,
directus   |       highWaterMark: 16384,
directus   |       buffer: BufferList { head: null, tail: null, length: 0 },
directus   |       length: 0,
directus   |       pipes: [],
directus   |       flowing: true,
directus   |       ended: false,
directus   |       endEmitted: false,
directus   |       reading: true,
directus   |       constructed: true,
directus   |       sync: false,
directus   |       needReadable: true,
directus   |       emittedReadable: false,
directus   |       readableListening: false,
directus   |       resumeScheduled: false,
directus   |       errorEmitted: false,
directus   |       emitClose: false,
directus   |       autoDestroy: true,
directus   |       destroyed: false,
directus   |       errored: null,
directus   |       closed: false,
directus   |       closeEmitted: false,
directus   |       defaultEncoding: 'utf8',
directus   |       awaitDrainWriters: null,
directus   |       multiAwaitDrain: false,
directus   |       readingMore: false,
directus   |       dataEmitted: true,
directus   |       decoder: null,
directus   |       encoding: null,
directus   |       [Symbol(kPaused)]: false
directus   |     },
directus   |     _events: [Object: null prototype] {
directus   |       end: [Function: onReadableStreamEnd],
directus   |       error: [Array],
directus   |       timeout: [Function],
directus   |       close: [Function],
directus   |       data: [Function (anonymous)]
directus   |     },
directus   |     _eventsCount: 5,
directus   |     _maxListeners: undefined,
directus   |     _writableState: WritableState {
directus   |       objectMode: false,
directus   |       highWaterMark: 16384,
directus   |       finalCalled: false,
directus   |       needDrain: false,
directus   |       ending: false,
directus   |       ended: false,
directus   |       finished: false,
directus   |       destroyed: false,
directus   |       decodeStrings: false,
directus   |       defaultEncoding: 'utf8',
directus   |       length: 0,
directus   |       writing: false,
directus   |       corked: 0,
directus   |       sync: false,
directus   |       bufferProcessing: false,
directus   |       onwrite: [Function: bound onwrite],
directus   |       writecb: null,
directus   |       writelen: 0,
directus   |       afterWriteTickInfo: null,
directus   |       buffered: [],
directus   |       bufferedIndex: 0,
directus   |       allBuffers: true,
directus   |       allNoop: true,
directus   |       pendingcb: 0,
directus   |       constructed: true,
directus   |       prefinished: false,
directus   |       errorEmitted: false,
directus   |       emitClose: false,
directus   |       autoDestroy: true,
directus   |       errored: null,
directus   |       closed: false,
directus   |       closeEmitted: false,
directus   |       [Symbol(kOnFinished)]: []
directus   |     },
directus   |     allowHalfOpen: false,
directus   |     _sockname: null,
directus   |     _pendingData: null,
directus   |     _pendingEncoding: '',
directus   |     server: null,
directus   |     _server: null,
directus   |     timeout: 0,
directus   |     [Symbol(async_id_symbol)]: 115,
directus   |     [Symbol(kHandle)]: TCP {
directus   |       reading: true,
directus   |       onconnection: null,
directus   |       [Symbol(owner_symbol)]: [Circular *1]
directus   |     },
directus   |     [Symbol(lastWriteQueueSize)]: 0,
directus   |     [Symbol(timeout)]: Timeout {
directus   |       _idleTimeout: -1,
directus   |       _idlePrev: null,
directus   |       _idleNext: null,
directus   |       _idleStart: 1028,
directus   |       _onTimeout: null,
directus   |       _timerArgs: undefined,
directus   |       _repeat: null,
directus   |       _destroyed: true,
directus   |       [Symbol(refed)]: false,
directus   |       [Symbol(kHasPrimitive)]: false,
directus   |       [Symbol(asyncId)]: 120,
directus   |       [Symbol(triggerId)]: 0
directus   |     },
directus   |     [Symbol(kBuffer)]: null,
directus   |     [Symbol(kBufferCb)]: null,
directus   |     [Symbol(kBufferGen)]: null,
directus   |     [Symbol(kCapture)]: false,
directus   |     [Symbol(kSetNoDelay)]: true,
directus   |     [Symbol(kSetKeepAlive)]: true,
directus   |     [Symbol(kSetKeepAliveInitialDelay)]: 0,
directus   |     [Symbol(kBytesRead)]: 0,
directus   |     [Symbol(kBytesWritten)]: 0
directus   |   },
directus   |   [Symbol(kCapture)]: false
directus   | }
directus   | return

### What version of Directus are you using?

9.14.5

### What version of Node.js are you using?

directus/directus:9 uses node 16

### What database are you using?

Postgres 13

### What browser are you using?

Chrome

### How are you deploying Directus?

Docker

**Comment by rijkvanzanten** [comment]
Odd! I don't think this is caused by a change to Directus, so lets see if we can find the culprit in the third party dependencies and/or Docker usage 🤔

**Comment by rijkvanzanten** [comment]
I think it might be in `@keyv/redis`. Docker is on `2.3.8`, my local (where I don't see the same issue) is on `2.3.7`.

**Comment by rijkvanzanten** [comment]
Yep, that seems to have been the culprit. There's a known issue in 2.3.8 that logs those things, that was fixed in 2.3.9. The next release of the docker image (coming tomorrow) should resolve that 👍🏻 

Ref https://github.com/jaredwray/keyv/issues/419

**Comment by hummal** [comment]
Thank you!
